ENIGMAsystem/SHELL/Universal_System/estring.cpp

```diff
@@ -172,6 +172,7 @@ double real(variant str) { return str.type ? atof(((string)str).c_str()) : (doub
 string ansi_char(char byte) { return string(1,byte); }
 string chr(char val) { return string(1,val); }
 int ord(string str)  { return str[0]; }
+int ord(char c) { return c; }
```

Comment:
In the game [Traffic Simulator](https://enigma-dev.org/edc/games.php?game=94), there was a script that called `ord` with characters, so I thought it would be better to have this overload.